fusor-ember-cli/bower.json

```diff
@@ -1,31 +1,27 @@
 {
   "name": "fusor-ember-cli",
   "dependencies": {
-    "ember": "1.13.10",
-    "ember-cli-shims": "ember-cli/ember-cli-shims#0.0.3",
-    "ember-cli-test-loader": "ember-cli-test-loader#0.1.3",
-    "ember-data": "1.13.14",
-    "ember-load-initializers": "ember-cli/ember-load-initializers#0.1.5",
-    "ember-qunit": "0.4.9",
-    "ember-qunit-notifications": "0.0.7",
-    "ember-resolver": "~0.1.18",
-    "jquery": "~1.11.3",
+    "ember": "~2.4.3",
+    "ember-cli-shims": "0.1.1",
+    "ember-cli-test-loader": "0.2.2",
+    "ember-qunit-notifications": "0.1.0",
     "jquery-csv": "google-code-export/jquery-csv#*",
-    "loader.js": "ember-cli/loader.js#3.2.1",
-    "qunit": "~1.18.0",
     "patternfly-sass": "2.10.0",
     "patternfly": "git://github.com/patternfly/patternfly#2.10.0",
     "ember-cli-moment-shim": "~0.1.0",
+    "es5-shim": "^4.0.5",
     "ip-address": "~5.8.0",
     "matchmedia": "0.2.0",
-    "pretender": "~0.9.0",
-    "lodash": "~3.7.0",
+    "filesize": "~3.3.0",
+    "font-awesome": "~4.5.0",
+    "pretender": "~0.10.1",
     "Faker": "~3.0.0",
-    "es5-shim": "^4.0.5",
     "filesize": "~3.3.0",
-    "bootstrap-switch": "~3.3.2"
+    "bootstrap-switch": "~3.3.2",
+    "blob-polyfill": "^1.0.20150320",
+    "lodash": "~3.7.0"
   },
   "devDependencies": {
-    "blanket": "~1.1.5"
+    "blanket": "5e94fc30f2e694bb5c3718ddcbf60d467f4b4d26"
```

Comment:
Just out of curiosity, where is blanket being used?  I saw it's not being maintained anymore.